client/tests/library/test_transpiler.py

```diff
@@ -31,8 +31,8 @@ def test_transpile(self):
         circuit1 = random_circuit(5, 3)
         circuit2 = random_circuit(5, 3)
 
-        backend1 = FakeAlmadenV2()
-        backend2 = FakeBrooklynV2()
+        backend1 = GenericBackendV2(num_qubits=5)
```

Comment:
Not sure it matters for these tests, but I found [this replacement list](https://github.com/Qiskit/qiskit/pull/10952) for deprecated fake providers, `FakeAlmadenV2` is now `Fake20QV1`.

Reply:
Hi @karlaspuldaro, 
I decided to use the `GenericBackendV2` because [it seems to be the only `qiskit.providers.fake_provider` that is not a legacy interface](https://docs.quantum.ibm.com/api/qiskit/providers_fake_provider).
Do you see an advantage of using `Fake20QV1` over `GenericBackendV2`?
I think either one would be fine 😃.